app/build.gradle

```diff
@@ -142,6 +142,10 @@ dependencies {
     implementation "com.google.android.gms:play-services-drive:17.0.0"
     implementation "com.google.android.gms:play-services-auth:19.0.0"
 
+    def work_version = "2.4.0"
+    implementation "androidx.work:work-runtime-ktx:$work_version"
+    implementation "androidx.work:work-gcm:$work_version"
+    androidTestImplementation "androidx.work:work-testing:$work_version"
 
```

Comment:
Since you are defining it here, can you make sure to delete the duplicate ones above? like in line 91

Reply:
Fixed in commit 885194f